tfx/proto/example_gen.proto

```diff
@@ -110,5 +110,7 @@ message SplitConfig {
   repeated Split splits = 1;
 
   // TODO(jyzhao): support feature based partition.
-  reserved 2;
+  string partition_feature_name = 2;
```

Comment:
follow that proto as much as possible, including the comments and structures

Reply:
I am not sure if we need to add ' oneof splitting_method_config ' in the proto.

Reply:
We don't need oneof I think